source/gameengine/Rasterizer/RAS_2DFilter.cpp

```diff
@@ -82,9 +83,18 @@ void RAS_2DFilter::Initialize(RAS_ICanvas *canvas)
 	}
 }
 
+int RAS_2DFilter::GetColorBindCode()
+{
+	return m_colorBindCode;
+}
+
 void RAS_2DFilter::Start(RAS_IRasterizer *rasty, RAS_ICanvas *canvas, unsigned short depthfbo,
 						 unsigned short colorfbo, unsigned short outputfbo)
 {
+	if (m_colorBindCode == -1) {
```

Comment:
As you can see in this function the off screen are switched very often, will the input off screen id fine for the user ?
Also this input off screen can be used as a output of the next filter (ping pong).